src/pages/ReportsUpload/UploadForm.tsx

```diff
@@ -52,7 +52,7 @@ class UploadForm extends React.Component<UploadFormProps, { }> {
             setTimeout(() => {
                 this.props.setFieldValue('file', file);
                 if (!this.props.values.reportName) {
-                    const fileNameWithoutExtension = file.name.replace(/\.[^/.]+$/, "");
+                    const fileNameWithoutExtension = file.name.replace(/(\.[^/.]+)+$/, "");
```

Comment:
In your example : myFile.whatever.tar.gz  I am understanding that "tar.gz" is the extension, not including the "whatever" that seems to be part of the name

So as I see your change is removing anything after the first dot ..... is that what is requested as "not include the extension of the file" ?

Reply:
@jonathanvila I think removing everything after the first dot is a more generic solution, so the UI can handle whatever extension the user selects e.g. ".zip, .tar.gz, .tar.xz, etc.". Having dots as part of the filename is not common (and in Windows is not allowed); in case the filename contains dots and the UI removes them, the user will always be able to edit the "Report name" field of the "Upload Form" page.

Reply:
ok